guides/source/configuring.md

```diff
@@ -1719,6 +1719,14 @@ config.active_storage.video_preview_arguments = "-vf 'select=eq(n\\,0)+eq(key\\,
 1. `select=eq(n\,0)+eq(key\,1)+gt(scene\,0.015)`: Select the first video frame, plus keyframes, plus frames that meet the scene change threshold.
 2. `loop=loop=-1:size=2,trim=start_frame=1`: To use the first video frame as a fallback when no other frames meet the criteria, loop the first (one or) two selected frames, then drop the first looped frame.
 
+#### `config.active_storage.multiple_file_field_include_hidden`
+
+In Rails 7.1 and beyond, Active Storage `has_many_attached` relationships will
+default to _replacing_ the current collection instead of _appending_ to it. To
+support submitting an _empty_ collection, render an auxiliary hidden field
+similar to how Action View Form Builder render checkbox elements. The default is
+`false`.
```

Comment:
@rafaelfranca I've added a Configuration Guide entry. Is there anything else I could add elsewhere?